src/all/holonometria/src/eu/kanade/tachiyomi/extension/all/holonometria/Holonometria.kt

```diff
@@ -112,13 +112,14 @@ class Holonometria(
         return chapters
```

Comment:
seems like the chapter list isn't paginated now, so you can remove this pagination logic

(should be fine just to remove the overrides of `chapterListRequest` and `chapterListParse`)